cli/command/container/create_test.go

```diff
@@ -189,8 +189,8 @@ func TestCreateContainerImagePullPolicyInvalid(t *testing.T) {
 
 			statusErr := cli.StatusError{}
 			assert.Check(t, errors.As(err, &statusErr))
-			assert.Equal(t, statusErr.StatusCode, 125)
-			assert.Check(t, is.Contains(dockerCli.ErrBuffer().String(), tc.ExpectedErrMsg))
+			assert.Check(t, is.Equal(statusErr.StatusCode, 125))
```

Comment:
Why this change?

Reply:
I think I changed the `assert.Equal` -> `assert.Check` here so that we don't fail early; `assert.Equal` will do a `t.Fatal()` if it fails (so fail the test immediately), whereas `assert.Check()` will do a `t.Error` (mark the test as failed, but continue)

Reply:
Ahh, sounds good!